platform/feature-api/communication/rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import sucrase from "@rollup/plugin-sucrase";
+import commonjs from "@rollup/plugin-commonjs";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import json from "@rollup/plugin-json";
+
+const communicationFileName = "src/index.ts";
+export default [
+    {
+        input: communicationFileName,
+        output: [
+            {
+                dir: "dist",
+                format: "esm",
+            },
+            /*            {
+                inlineDymamicImports: true,
+                file: "dist/index.js",
+                format: "cjs",
+            },
+            {
+                file: "dist/communication.js",
+                format: "iife",
+                name: "communication",
+            },
+ */
```

Comment:
Can you maybe write a few words about _why_ this is commented out? Ideally just remove this stuff if there's no reason for us to still need it.

Reply:
Right, I should just remove it.